### ISSUE TITLE
Client not downloading mods

### ISSUE THREAD

**Issue by osum4est** [superzanti/ServerSync#26]
Hello, I'm using version 2.3 of this mod, and the client isn't downloading the mods from the server. Both the client and the server have the same config file, and I've made sure both ports are forwarded to the server. When the client uses the mod, it attempts to connect to the server but then fails due to mod rejections. Am I doing something wrong? 

Here's my config: 
```
# Configuration file

gui {
    # The ID of the button that connects to the server and updates [range: 0 ~ 2147483647, default: 6001]
    I:ButtonID=6003
}


ignoredfiles {
    # These files are ignored by serversync. DO NOT IGNORE serversync.cfg
    S:IGNORE_LIST <
        ./mods/CustomMainMenu-MC1.7.10-1.6.jar
        ./mods/DynamicLights-1.7.10.jar
        ./mods/DefaultWorldGenerator-1.7.10-0.1-b13-universal.jar
        ./config/CustomMainMenu/mainmenu.json
        ./config/forge.cfg
        ./config/forgeChunkLoading.cfg
        ./config/splash.properties
     >
}


serverconnection {
    # The port in which the minecraft server is running, not the serversync port [range: 1 ~ 49151, default: 25565]
    I:MINECRAFT_PORT=11200

    # The IP address of the server [default: 127.0.0.1]
    S:SERVER_IP=174.52.155.148

    # The port that your server will be serving on [range: 1 ~ 49151, default: 38067]
    I:SERVER_PORT=11100
}


serverencryption {
    # The check command security key phrase [default: 0ba4439ee9a46d9d9f14c60f88f45f87]
    S:SECURE_CHECK=0ba4439ee9a46d9d9f14c60f88f45f87

    # The checksum command security key phrase [default: 226190d94b21d1b0c7b1a42d855e419d]
    S:SECURE_CHECKSUM=226190d94b21d1b0c7b1a42d855e419d

    # The exists command security key phrase [default: e087923eb5dd1310f5f25ddd5ae5b580]
    S:SECURE_EXISTS=e087923eb5dd1310f5f25ddd5ae5b580

    # The exit command security key phrase [default: f24f62eeb789199b9b2e467df3b1876b]
    S:SECURE_EXIT=f24f62eeb789199b9b2e467df3b1876b

    # The recursive command security key phrase [default: f8e45531a3ea3d5c1247b004985175a4]
    S:SECURE_RECURSIVE=f8e45531a3ea3d5c1247b004985175a4

    # The update command security key phrase [default: 3ac340832f29c11538fbe2d6f75e8bcc]
    S:SECURE_UPDATE=3ac340832f29c11538fbe2d6f75e8bcc
}


storagevariables {
    # DO NOT EDIT THIS LINE UNLESS YOU KNOW WHAT YOU ARE DOING! (If you are a server feel free to change it as much as you want to update your clients) [default: 20150608_000500]
    S:LAST_UPDATE=20150608_000500
}



```

**Comment by osum4est** [comment]
Solved my problem by updating to version 2.4. Sorry, should've checked that first. If you could put 2.4 on curseforge that'd be awesome! :)

**Comment by superzanti** [comment]
I will do that soon. Thank you.
